utils/utils.jsx

```diff
@@ -207,7 +207,7 @@ export function ding() {
 }
 
 export function hasSoundOptions() {
-    return (!(UserAgent.isFirefox()) && !(UserAgent.isEdge()));
+    return (!(UserAgent.isEdge()));
```

Comment:
what would be the user agent for chromium based edge? would this work?

Reply:
Looking in [`utils/user_agent.tsx`](https://github.com/mattermost/mattermost-webapp/blob/master/utils/user_agent.tsx#L8) we see these two example UAs for chrome and edge respectively:

```
Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2704.103 Safari/537.36
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/46.0.2486.0 Safari/537.36 Edge/13.10586
```

Where it gets interesting is looking at the UA string for the new chromium edge:

```
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.122 Safari/537.36 Edg/80.0.361.62
```

And when we look at the [`isChrome()`](https://github.com/mattermost/mattermost-webapp/blob/master/utils/user_agent.tsx#L51) check, because of the missing "e" in new edge UA (`Edg/80.0.361.62`) it will actually match as a chrome:

```
export function isChrome(): boolean {
    return userAgent.indexOf('Chrome') > -1 && userAgent.indexOf('Edge') === -1;
}
```

I think this is a good thing in as much as our UA detection will think it's chrome.  But it may be a bad thing because it's some hw